docs/applications/configuration-management/Using-Laravel-Forge-with-Linode.md

```diff
@@ -0,0 +1,226 @@
+
+When the server is active, you will also receive mail from Linode with details of things that has been created.
+
+[![Linode API](/docs/assets/configuration-management/mail_from_linode_showing_details_created.png)](/docs/assets/configuration-management/mail_from_linode_showing_details_created.png)
+
+You will also receive a mail that a public key was added to your gitlab account.
+
+Now, lets confirm that the server is created on Linode also, head over to linode under the linode tab, you should see a node that has been created.
+
+
+[![Linode API](/docs/assets/configuration-management/linode_created.png)](/docs/assets/configuration-management/linode_created.png)
+
+
+## Pointing your domain name
+
+If you intend to visit the site you're setting up via a real domain (versus just testing it by visiting the IP address), you have to add the domain because DNS record changes can take a while to update. 
+
+1. Now head back to Laravel Forge account then click on the active server. 
+
+2. Go to your domain name provider, add an ANAME record for the domain or subdomain, and point it to the public IP address of your server. Your public IP address is as seen in the screenshot below, mine is: `41.33.92.166` That's it!
+
+
+[![Linode API](/docs/assets/configuration-management/adding_new_domain.png)](/docs/assets/configuration-management/adding_new_domain.png)
+
+- Root Domain: This is your domain name.
+- Project Type: This is the type of of project you are using, If you are building regular php, you can choose : ` General PHP / Laravel`. ( the other option available are : General PHP / Laravel, Static, HTML, Symfony, Symfony (Dev).       
+- Web Directory: This is where you public files will be served from.
+
+3. Submit the form by clicking `ADD SITE`.
+
+
+{: .note}
+>
+> By default, a default site is created for you, so if you don’t want to add a domain name now, you can visit your public IP which is linked to the Default site created.
+
+
+## Adding our repository and auto-deployment
+
+1. Click on your active servers then under the active site, choose the domain name you added. If you are just testing out, then click on the `Default`.
+
+[![Linode API](/docs/assets/configuration-management/adding_repository.gif)](/docs/assets/configuration-management/adding_repository.gif)
+
+2. Click on Apps on the left hand side and then select your `Git Repository`.
+
+[![Linode API](/docs/assets/configuration-management/adding_git_repository.png)](/docs/assets/configuration-management/adding_git_repository.png)
+
+3. Fill in the repository field with your username and repository then select the branch you want to use. Now, click on INSTALL REPOSITORY. Take a closer look how the `Repository` Field is inserted which is  gitlab_username/repository_name_without_.git`. 
+
+eg: 
+
+```
+    dongido/transaction-manager
+
+```
+
+[![Linode API](/docs/assets/configuration-management/git_repository_name.png)](/docs/assets/configuration-management/git_repository_name.png)
```

Comment:
Remove personal identifying repository name. Update remove yellow highlighted marker. Replace with rectangular selection tool per examples in previous guides: https://www.linode.com/docs/platform/api/api-key